release-notes/cve-index.json

```diff
@@ -0,0 +1,21 @@
+{
+    "$schema": "https://json.schemastore.org/dotnet-cve-index.json",
+    "cve-index": [
+        {
```

Comment:
9.0 is empty and should not be listed until there is at least one CVE in any version belonging to `9.0`.